solutions/landing-zone-v2/namespaces/projects.yaml

```diff
@@ -111,6 +111,23 @@ spec:
   role: roles/serviceusage.serviceUsageAdmin
   member: "serviceAccount:projects-sa@management-project-id.iam.gserviceaccount.com" # kpt-set: serviceAccount:projects-sa@${management-project-id}.iam.gserviceaccount.com
 ---
+# Grant GCP role compute XPN Admin to GCP SA
```

Comment:
that role was already granted to the networking namespace/service account. Can the networking namespace be used instead ?

Reply:
done , removed from projects